apps/production/views.py

```
@@ -1297,9 +1245,148 @@ def _default_warehouse(order=None):
+                    canteen=document.canteen,
+                    date=day_date,
+                    meal_type=ProductionOrder.MealType.DINNER_SECOND,
+                ).first()
+                if order_obj is None:
+                    sibling = ProductionOrder.objects.filter(
+                        picking_list_items__document=document, date=day_date
+                    ).first()
+                    if sibling is None:
+                        messages.error(
+                            request,
+                            f'Pro den {day_date.strftime("%d.%m.%Y")} nelze založit druhou večeři – den nemá žádné jídlo.'
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)